### ISSUE TITLE
top-level groups not reported

### ISSUE THREAD

**Issue by egonelbre** [tetafro/godot#8]
Top level grouped items such as `const`, `var`, `type` are not being reported.

```
package main

const (
	// Alpha is one
	Alpha = 1
)

var (
	// Beta is two
	Beta = 2
)

type (
	// Gamma is a struct
	Gamma struct{}
)
```

**Comment by tetafro** [comment]
Yeah, you're right. I'll fix it. Thanks for report.

**Comment by tetafro** [comment]
Fixed in 0.4.5.

It will appear in golangci-lint shortly - dependabot updates deps there.
